examples/passport-login/package.json

```diff
@@ -0,0 +1,91 @@
+{
+  "name": "@loopback/passport-login",
+  "version": "1.0.0",
```

Comment:
Let's name it as `@loopback/example-passport-login` to be consistent with other example projects.

Reply:
agreed, modified now as suggested